### ISSUE TITLE
No returns from requests

### ISSUE THREAD

**Issue by tobibuchmann** [docker/genai-stack#86]
Everything seems to start smoothly, but I don't get any returns wehen I use the different apps. It says for instance "Model: llama2:13b
RAG: Disabled" but no further response when I ask "How can I calculate age from date of birth in Cypher?" what could be the issue here?

**Comment by jexp** [comment]
Yes you need to use 4.24.1 at least, there was a bug in Docker in between that broke pytorch.

If you can enable langsmith you should be able to see what's happening behind the scenes or use `verbose=True` for the LLM or chains.

**Comment by tobibuchmann** [comment]
Thanks for your ideas. Unfortunately I was't able to solve the issue yet. I upgrade from Docker 4.23 to 4.25 but that didn't solve it. I don't have access to langsmith. Where would be the best way to use verbose = True? Or what is the best way to debug? Which logs should I check?
Thanks
BTW, this is from my VS Code terminal:
genai-stack-api-1         | INFO:     127.0.0.1:52148 - "GET / HTTP/1.1" 200 OK
genai-stack-bot-1         | 2023-11-06 15:07:32.688 Embedding: Using SentenceTransformer
genai-stack-bot-1         | 2023-11-06 15:07:32.701 LLM: Using Ollama: llama2:13b
genai-stack-api-1         | INFO:     127.0.0.1:48336 - "GET / HTTP/1.1" 200 OK
genai-stack-bot-1         | 2023-11-06 15:07:50.575 Embedding: Using SentenceTransformer
genai-stack-bot-1         | 2023-11-06 15:07:50.590 LLM: Using Ollama: llama2:13b
genai-stack-api-1         | INFO:     127.0.0.1:54948 - "GET / HTTP/1.1" 200 OK
genai-stack-bot-1         | 2023-11-06 15:08:16.286 Embedding: Using SentenceTransformer
genai-stack-bot-1         | 2023-11-06 15:08:16.299 LLM: Using Ollama: llama2:13b
genai-stack-api-1         | INFO:     127.0.0.1:39352 - "GET / HTTP/1.1" 200 OK
genai-stack-bot-1         | 2023-11-06 15:08:40.502 Embedding: Using SentenceTransformer
genai-stack-bot-1         | 2023-11-06 15:08:40.516 LLM: Using Ollama: llama2:13b
genai-stack-bot-1         | 2023-11-06 15:08:44.372 Embedding: Using SentenceTransformer
genai-stack-bot-1         | 2023-11-06 15:08:44.386 LLM: Using Ollama: llama2:13b
genai-stack-api-1         | INFO:     127.0.0.1:37368 - "GET / HTTP/1.1" 200 OK
genai-stack-api-1         | INFO:     127.0.0.1:60360 - "GET / HTTP/1.1" 200 OK
genai-stack-api-1         | INFO:     127.0.0.1:60272 - "GET / HTTP/1.1" 200 OK
genai-stack-bot-1         | 2023-11-06 15:22:55.763 Embedding: Using SentenceTransformer
genai-stack-bot-1         | 2023-11-06 15:22:55.797 LLM: Using Ollama: llama2:13b
genai-stack-api-1         | INFO:     127.0.0.1:49086 - "GET / HTTP/1.1" 200 OK

**Comment by slimslenderslacks** [comment]
@tobibuchmann is this one resolved for you now?
